packages/e2e-tests/specs/experiments/navigation-editor.test.js

```diff
@@ -328,68 +282,4 @@ describe( 'Navigation editor', () => {
 		} );
 		expect( submenuLinkHidden ).toBeDefined();
 	} );
-
-	it( 'displays suggestions when adding a link', async () => {
```

Comment:
Instead of deleting the tests, could be switch to `it.skip( ... )` to keep the context. I know source control history could always be browsed, but once out of sight it will be out of mind.